hfst-language-module/Makefile.am

```diff
@@ -111,8 +111,8 @@ debug: .deps/$(LANG1).LR-debug.hfst .deps/$(LANG1).lexc.hfst
 ## Disambiguation rules
 ###############################################################################
 
-$(LANG1).rlx.bin: $(BASENAME).$(LANG1).rlx $(CGCOMP)
-	$(CGCOMP) $< $@
+$(LANG1).rlx.bin: $(BASENAME).$(LANG1).rlx
```

Comment:
Why is this not necessary?

Reply:
`$(CGCOMP)` comes from the test that finds the cg-comp binary, but when using pkg-config one can rely on cg-comp existing and `$(CGCOMP)` isn't set.